app/views/posts/index.html.erb

```diff
@@ -13,3 +13,14 @@
 
     <%= submit_tag 'Submit' %>
   <% end %>
+
+  <div class="ghost-container">
+    <div class="ghost-body">
+      <div class="eyeball"></div>
+      <div class="eyeball"></div>
+    </div>
+    <div class="flap"></div>
+    <div class="flap"></div>
+    <div class="flap"></div>
+    <div class="shadow"></div>
+  </div>
```

Comment:
I would move this block into the post section (between lines 6 and 7) - that way, every individual post will have its own ghost